src/pages/orthomosaic_display/App.js

```diff
@@ -1,116 +1,152 @@
-import { useState } from 'react';
+import { useEffect, useState } from 'react';
```

Comment:
* it might make sense to rename some of these files like `orthomosaic_display/App.js` and flightListSideBar to make it more suitable to the current pages. 
* And also move header to "components"